src/components/customers/AddWalletToCustomerDialog.tsx

```diff
@@ -0,0 +1,49 @@
+import { forwardRef } from 'react'
```

Comment:
I wouldn't add this file to this PR to be honest. A lot of things are missing and I won't work on it so you can keep it separate

Reply:
Agree, it's been removed